### PR TITLE
Update Tier System Icons and Formatting

### PR DESCRIPTION
**Pull request by mativanarquero**
launch_ss
launch_ss_grid

### CODE REVIEW

**Review by swengr-janan — CHANGES_REQUESTED** [review]
image
Pa uppercase lahat ng letters @mativanarquero

File: src/pages/Home/AboutSection/config.ts [hunk 1]
```diff
@@ -39,21 +39,21 @@ export const TierSystemList = [
         title: 'IGNITE',
         requirement: '100,000 SRKb/ 6070 SFUEL Staked',
         poolWeight: 10,
-        guaranteedAllocation: 'yes',
+        guaranteedAllocation: 'Yes',
```
Comment (kbaldo) [inline src/pages/Home/AboutSection/config.ts:42]: @mativanarquero, kuya @swengr-janan requested to capitalize all the letters. So far, it seems like yung `Y` palang from `Yes` is capitalized haha

Comment (swengr-janan) [reply src/pages/Home/AboutSection/config.ts:42]: Yep, ilahat mo na @mativanarquero .

Comment (mativanarquero) [reply src/pages/Home/AboutSection/config.ts:42]: Oops! Done did it, will push first thing tomorrow morning. Thank you.

Comment (mativanarquero) [reply src/pages/Home/AboutSection/config.ts]: Done po sir. Pacheck nalang po. Thank you.

Comment (swengr-janan) [reply src/pages/Home/AboutSection/config.ts]: AGITH!

**Review by swengr-janan — APPROVED** [review]
All goods!

**Pull request merged**